src/app/browser/qgsinbuiltdataitemproviders.cpp

```diff
@@ -1060,6 +1061,73 @@ void QgsDatabaseItemGuiProvider::populateContextMenu( QgsDataItem *item, QMenu *
         menu->addAction( newTableAction );
       }
     }
+
+    // SQL dialog
+    if ( std::unique_ptr<QgsAbstractDatabaseProviderConnection> conn( item->databaseConnection() ); conn && conn->capabilities().testFlag( QgsAbstractDatabaseProviderConnection::Capability::ExecuteSql ) )
+    {
+      QAction *sqlAction = new QAction( QObject::tr( "Run SQL command…" ), menu );
```

Comment:
```suggestion
      QAction *sqlAction = new QAction( QObject::tr( "Run SQL Command…" ), menu );
```

Reply:
(or what do you think about just "Execute SQL..."? "Command" still doesn't sit right with me.)

Reply:
Deal